parsing/ast_helper.mli

```diff
@@ -369,7 +369,7 @@ module Incl:
 module Vb:
   sig
     val mk: ?loc: loc -> ?attrs:attrs -> ?docs:docs -> ?text:text ->
-      ?typ:poly_constraint -> pattern -> expression -> value_binding
+      ?typ:value_constraint -> pattern -> expression -> value_binding
```

Comment:
is `?typ` an appropriate label/field name now that this is a more generic annotation?

Reply:
`?value_constraint`?